edit.js

```diff
@@ -1631,8 +1631,10 @@ const _changeInventory = inventory => {
       loginManager.setAvatar(dataHash);
     });
     const removeButton = itemEl.querySelector('.remove-button');
-    removeButton.addEventListener('click', () => {
+    removeButton.addEventListener('click', async () => {
       console.log('remove', item);
+      const inventory = loginManager.getInventory().filter(i => i.dataHash !== item.dataHash);
```

Comment:
Do we need to `getInventory`? It looks like we might have it as `inventory` above.

Reply:
Thanks, I didn't spot that! Updated :)